README.md

```diff
@@ -3,9 +3,11 @@ KestrelHttpServer
 
 [![Join the chat at https://gitter.im/aspnet/KestrelHttpServer](https://badges.gitter.im/aspnet/KestrelHttpServer.svg)](https://gitter.im/aspnet/KestrelHttpServer?utm_source=badge&utm_medium=badge&utm_campaign=pr-badge&utm_content=badge)
 
-AppVeyor: [![AppVeyor](https://ci.appveyor.com/api/projects/status/nr0s92ykm57q0bjv/branch/dev?svg=true)](https://ci.appveyor.com/project/aspnetci/KestrelHttpServer/branch/dev)
+[![Travis build status][travis-badge]](https://travis-ci.org/aspnet/KestrelHttpServer/branches)
```

Comment:
IMO the branches page is a better view of build status.

![image](https://user-images.githubusercontent.com/2696087/29425761-497e739c-8339-11e7-917f-28ea3ca99fc5.png)


The default page can be misleading because it shows the most recent build, which could be a failing PR or feature branch.